source/operating-a-service/incident-playbook.html.md

```diff
@@ -64,43 +73,47 @@ Contact your users if:
 
 Informing users about incidents is generally considered best practice, but should be decided on a case by case basis with the product and service managers.
 
-### 8. If the incident requires invocation of Disaster Recovery procedures
+The Teacher Services team maintains a publicly available [service status dashboard](https://teacher-services-status.education.gov.uk/). For some incidents, you might find it helpful to update this. You will need a GitHub account to do this, or delegate updates to a colleague who has one.
```

Comment:
```suggestion
The Teacher Services team maintains a publicly available [service status dashboard](https://teacher-services-status.education.gov.uk/), generated by our [Upptime](https://github.com/DFE-Digital/teacher-services-upptime) instance on GitHub.
```

Reply:
The updates are automatic

Reply:
Oh wow! it didn't say that before - said the comms lead needed to update it!